packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -60,20 +59,27 @@ class _DebuggerConsoleState extends State<DebuggerConsole> {
 
     return OutlineDecoration(
       child: Console(
-        title: debuggerSectionTitle(Theme.of(context), text: 'Console'),
-        lines: _lines,
-        controls: [
-          CopyToClipboardControl(
-            dataProvider: disabled ? null : () => _lines.join('\n'),
-            successMessage: 'Copied $numLines ${pluralize('line', numLines)}.',
-            buttonKey: DebuggerConsole.copyToClipboardButtonKey,
-          ),
-          DeleteControl(
-            onPressed: disabled ? null : widget.controller.clearStdio,
-            tooltip: 'Clear console output',
-            buttonKey: DebuggerConsole.clearStdioButtonKey,
+        title: areaPaneHeader(
+          context,
+          title: 'Console',
+          needsTopBorder: false,
+          rightChild: Row(
```

Comment:
`rightChild` can maybe be renamed `children` and it becomes a `<Widget>[]`? That way the AreaPaneHeader controls how it wants to render these children (instead of having a Row of things).

I can see this becoming a thing in a "collapsed" state of the toolbar, if not all the buttons fit in the header area and you want to render some underneath a "..."

(Like the toolbar in sheets)
<img width="462" alt="Screen Shot 2020-06-05 at 4 07 02 PM" src="https://user-images.githubusercontent.com/1255594/83929002-a53cdc80-a746-11ea-8ec1-e128d1deea39.png">

Reply:
Hmm, perhaps `rightChildren`? `actions`? We may want some name other than `child` given that there's other content being specified here as well (title).

Reply:
both `rightChildren` or `actions` sound good to me!